containers/FlyBox.js

```diff
@@ -48,17 +50,24 @@ const FlatListItemSeparator = () => {
     )
 }
 
+class FlyBox extends Component {
+    constructor(props) {
+        super(props);
+    }
+
+    async componentDidMount() {
+        this.props.setFlies(TestFlies)
```

Comment:
Thanks for adding this componentDidMount! Like we talked about on our call, we can then add the API fetch request on line 58.5 and do a .then for the this.props.setFlies and setFlies using the retrieved data.